src/components/HeaderWithCloseButton.js

```diff
@@ -138,26 +165,37 @@ class HeaderWithCloseButton extends Component {
                     styles.overflowHidden,
                 ]}
                 >
-                    {this.props.shouldShowBackButton && (
-                        <Tooltip text={this.props.translate('common.back')}>
-                            <Pressable
-                                onPress={() => {
-                                    if (this.props.isKeyboardShown) {
-                                        Keyboard.dismiss();
-                                    }
-                                    this.props.onBackButtonPress();
-                                }}
-                                style={[styles.touchableButtonImage]}
-                            >
-                                <Icon src={Expensicons.BackArrow} />
-                            </Pressable>
-                        </Tooltip>
+                    <View style={[styles.flexRow, styles.alignItemsCenter]}>
+                        {this.props.shouldShowBackButton && (
+                            <Tooltip text={this.props.translate('common.back')}>
+                                <Pressable
+                                    onPress={() => {
+                                        if (this.props.isKeyboardShown) {
+                                            Keyboard.dismiss();
+                                        }
+                                        this.props.onBackButtonPress();
+                                    }}
```

Comment:
I know you didn't write this but should this be extracted into a `useCallback` or `useMemo` hook, I think yes but not 100% sure.

Reply:
Wouldn't we do that when we refactor this component to use Hooks?

Reply:
oh doh, i thought this was but must not have realized I moved into a different component when scrolling down the review window 🤦